IntegrationTests/GoldMaster/Bolts.podspec.json.goldmaster

```diff
@@ -107,25 +107,25 @@ filegroup(
   name = "Tasks_hdrs",
   srcs = select(
     {
-      ":watchosCase": glob(
+      "//conditions:default": glob(
```

Comment:
We need to solve this unstable output at some point, do we have an issue out for this?

Reply:
+1 adding a `help wanted` to #41  😺